xlb/operator/stepper/nse_stepper.py

```diff
@@ -40,6 +40,9 @@ def __init__(self, omega, boundary_conditions=[], collision_type="BGK", forcing_
 
         operators = [self.macroscopic, self.equilibrium, self.collision, self.stream]
 
+        self.boundary_conditions = boundary_conditions
```

Comment:
`self.boundary_conditions = boundary_conditions` is already in the parent class. please remove this